server/src/models/UserPersonalization.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryBuilder } from 'objection';
+
+import { Domain } from './Domain';
+import { User } from './User';
+import { withDates } from './utils';
+
+export class UserPersonalization extends withDates(Domain) {
+  id!: string;
+  user_id!: string;
+
+  type!: string | null;
+  value!: string | null;
+
+  user!: User;
+
+  static tableName = 'omscentral_user_personalization';
+
+  static relationMappings = {
+    user: {
+      relation: Domain.BelongsToOneRelation,
+      modelClass: UserPersonalization,
+      join: {
+        from: `${UserPersonalization.tableName}.user_id`,
+        to: `${User.tableName}.id`,
+      },
+    },
+  };
+
+  static jsonSchema = {
+    type: 'object',
+    required: ['id', 'user_id'],
+    properties: {
+      id: { type: 'string' },
+      user_id: { type: 'string' },
+      type: { type: ['string', 'null'] },
+      value: { type: ['string', 'null'] },
+      user: User.jsonSchema,
+    },
+  };
+
+  static eagerQuery = (): QueryBuilder<UserPersonalization> =>
```

Comment:
a sanity check
:does this eager query look right?

My understanding:
Basically, based on  usages in current code, ```[user]`` will find the user corresponding to user personalization.

  Quoting [documentation](https://vincit.github.io/objection.js/api/query-builder/eager-methods.html#withgraphfetched) " withGraphFetched is actually much much faster in some cases where the relationExpression contains multiple many-to-many or **has-many** relations" 
Specifically, the reason a eager query makes sense in this case is because user to user personalization is 1: many and so we get the performance benefit mentioned above.